lib/src/main/java/com/cloudinary/android/uploadwidget/ui/imagepreview/UploadWidgetImageView.java

```diff
@@ -174,14 +186,49 @@ public void cropImage() {
         cropOverlayView.setVisibility(INVISIBLE);
     }
 
+    /**
+     * Rotate the image by 90 degrees.
+     */
+    public void rotateImage() {
+        rotationAngle += 90 % 360;
```

Comment:
That should be `(rotationAngle += 90) % 360`.
Since `%` has precedence over `+=` the right hand side will always be 90 (`90 % 360`), `rotationAngle` will be cumulative and won't reset at 360.